consensus/src/types/mod.rs

```diff
@@ -242,15 +253,31 @@ pub struct FinalityUpdate {
     pub signature_slot: U64,
 }
 
-#[derive(serde::Deserialize, Debug)]
+#[derive(serde::Deserialize, serde::Serialize, Debug, Default)]
+pub struct FinalityUpdateSerde {
+    pub attested_header: Header,
+    pub finalized_header: Header,
+    pub finality_branch: Vec<Bytes32>,
+    pub sync_aggregate: SyncAggregate,
+    pub signature_slot: U64,
+}
```

Comment:
What is the purpose of this struct? 

1. name is unclear
2. no documentation
3. seems like a duplication of another struct with added fields. If you're adding a struct that's similar to another one, please explain why it is necessairy - what was lacking in the previous struct?